fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_flaws.dm

```diff
@@ -89,9 +99,9 @@
 		Tremere - Burn in the Chapel, Vassal Mutilation.<br> \
 		Ventrue - Cant drink from mindless mobs, can't level up, raise a vassal instead.<br></span>")
 	if(!isbeefman(bloodsucker))
-		to_chat(owner, "<span class='announce'>Malakavian - Hallucinations and Bluespace prophet.<br></span>")
-	to_chat(owner, "<span class='announce'>* Read more about Clans here: https://wiki.fulp.gg/en/Bloodsucker.<br></span>")
-
+		to_chat(owner, "<span class='announce'>Malkavian - Hallucinations and Bluespace prophet.<br></span>")
+	to_chat(owner, "<span class='announce'>You can find details on each Clan on the Bloodsuckers wiki page here:<br> \
+				* https://wiki.fulp.gg/en/Bloodsucker</span>")
```

Comment:
im not sure if you guys have access to config, but this should use policy instead of being hardcoded, read https://github.com/tgstation/tgstation/blob/master/.github/POLICYCONFIG.md